<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="accessibleName">
    <string>login_username</string>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="rememberPsd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>258</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLogin">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>258</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>284</y>
     <width>194</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>194</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>194</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreAccountLogin">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>300</y>
     <width>23</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>19</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loginState">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>254</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonRegistAccount">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>193</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>注册帐号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonForgetPassword">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>220</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="userHead">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtongFlicker">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>300</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonArrow">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>userHead</zorder>
  <zorder>loginState</zorder>
  <zorder>accountComboBox</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>rememberPsd</zorder>
  <zorder>autoLogin</zorder>
  <zorder>loginButton</zorder>
  <zorder>moreAccountLogin</zorder>
  <zorder>pButtonRegistAccount</zorder>
  <zorder>pButtonForgetPassword</zorder>
  <zorder>pButtongFlicker</zorder>
  <zorder>pButtonArrow</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
